.github/workflows/gradle-build-tag-publish-package.yaml

```diff
@@ -0,0 +1,91 @@
+name: 'Build, Tag and Publish gradle package'
```

Comment:
Er interesert i innspill på filenavn og navn.

Reply:
Tenker at filnavn er helt ok for nå. Tror vi blir nødt til å splitte ut spesifikke workflows i egene repos for å kunne gjøre fornuftig version tagging.